### PR TITLE
Updated the vscode settings jason files/license

### PR DESCRIPTION
**Pull request by Jerry-Jinfeng-Guo**
Updated the license information of the vscode settings json files and moved the license information to license files

Follow-up to #481 + #483

### CODE REVIEW

File: .vscode/extensions.json.license [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+
+SPDX-License-Identifier: MPL-2.0
```
Comment (mgovers) [inline .vscode/extensions.json.license:3]: please add a whilteline at the EOF for consistency

File: .vscode/settings.json.license [hunk 2]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+
+SPDX-License-Identifier: MPL-2.0
```
Comment (mgovers) [inline .vscode/settings.json.license:3]: please add a whilteline at the EOF for consistency

**Comment by mgovers** [comment]
CI fails probably due to borked runner image or something

File: .vscode/settings.json.license (hunk 2 above)
Comment (Jerry-Jinfeng-Guo) [reply .vscode/settings.json.license]: Included in the latest commit.

File: .vscode/extensions.json.license (hunk 1 above)
Comment (Jerry-Jinfeng-Guo) [reply .vscode/extensions.json.license]: Included in the latest commit.

**Review by mgovers — APPROVED** [review]

**Pull request merged**